src/urllib3/multipart/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""Multipart support for urllib3."""
+
+from .decoder import (
```

Comment:
Should we hide the encoder/decoder submodules and reassign the names in this module? (Ie from .encoder import x as x)